deploy/metrics.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: allocation-api
+  namespace: offender-management-staging
+  labels:
+    k8s-apps: allocation-api
+spec:
+  jobLabel: allocation-api
+  selector:
+    matchExpressions:
+    - {key: allocation-api, operator: Exists}
```

Comment:
This should be able to select a `Service` in your namespace.

Reply:
@alkar - Matching service definition is here - https://github.com/ministryofjustice/offender-management-allocation-api/blob/8aa8e94277d129980dfb3cc443228b8169fad33c/deploy/service.yaml#L12